backend/Alpaki.Logic/Handlers/UpdateDreamer/DreamNotFoundException.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Alpaki.Logic.Handlers.UpdateDreamer
+{
+    public class DreamNotFoundException : LogicException
+    {
+        public override string Code => "DREAM_NOT_FOUND";
+
+        public DreamNotFoundException(long dreamId) : base($"Dream with given DreamId does not exists [DreamId={dreamId}]")
```

Comment:
hehe :) sam mówiłeś że zwracamy po polsku komunikaty :D poza tym. zrób to w walidatorze. zobacz GetDreamRequestValidator. jeśli coś zrobił źle user to jego bląd 4** jeśli coś zrobiliśmy źle my to 5**. tu zrobić źle user